neqo-http3/src/connection.rs

```diff
@@ -192,6 +192,15 @@ impl ::std::fmt::Display for Http3Connection {
     }
 }
 
+macro_rules! transaction {
+    ($t:expr) => {
+        match $t {
+            Some(t) => t,
+            _ => return Err(Error::InvalidStreamId),
```

Comment:
don't think you need this macro. Instead add `.ok_or_else(|| Error::InvalidStreamId)?`

Reply:
Or `.ok_or(Error::InvalidStreamId)`.